app/deployments/tasks.py

```diff
@@ -1,9 +1,16 @@
+from datetime import datetime, timedelta, timezone
 import logging
 
 from celery import shared_task
+import requests
```

Comment:
Codacy found an issue: [Unable to import 'requests' (import-error)](https://app.codacy.com/manual/gmri/buoy_barn/pullRequest?prid=6285790)